tests/functional/CSV/FingersCrossed/LoaderTest.php

```diff
@@ -60,4 +62,11 @@ public function testLoadCsvSuccessful()
             new Loader($this->writer)
         );
     }
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
+        );
+    }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)